### PR TITLE
Migrate configuration and installation steps to Ansible

### PR DESCRIPTION
**Pull request by yussufsh**

### CODE REVIEW

File: ansible/playbooks/roles/ocp-config/tasks/main.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    dest: "{{ workdir }}/install-config.yaml.back"
+    remote_src: yes
+    force: yes
+
+- name: Generate manifest files
+  shell: "openshift-install create manifests --log-level {{ log_level }}"
+  args:
+    chdir: "{{ workdir }}"
+
+- name: Remove the machines yaml files
```
Comment (bpradipt) [inline ansible/playbooks/roles/ocp-config/tasks/main.yaml]: machines => machine

File: ansible/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# Extra Ansible Playbooks for OCP4 Install
+
+These playbooks are used for installation of OCP on PowerVC and other post installation steps. Before running these playbooks on bastion node, ensure that all the required services are configured. Also, the cluster nodes needs to be booted on PowerVC.
+
+## Assumptions:
+
+ - A bastion node is already created where these playbooks would run.
+ - The [ocp4-helpernode](https://github.com/RedHatOfficial/ocp4-helpernode) playbook is executed once on the bastion node.
+ - The cluster nodes are already created on PowerVC manually, or by using IaC tools such as Terraform.
+ - These playbook are supported to use with https://github.com/ppc64le/ocp4_upi_powervm only.
```
Comment (bpradipt) [inline ansible/README.md]: playbook => playbooks

File: ansible/playbooks/roles/ocp-config/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+ocp-config: OCP Configuration
+=========
+
+This module will create the ignition files for the cluster nodes. All the ignition files will be hosted at `/var/www/html/ignition/`.
+
+Requirements
+------------
+
+ - The [ocp4-helpernode](https://github.com/RedHatOfficial/ocp4-helpernode) playbook should be executed once on bastion node.
```
Comment (bpradipt) [inline ansible/playbooks/roles/ocp-config/README.md]: Why this specific constraint ?

Comment (yussufsh) [reply ansible/playbooks/roles/ocp-config/README.md]: Let me rephrase it to "All the required configurations are done on the bastion node eg: HTTP, openshift-install binary is available on $PATH. This can also be achieved by using  [ocp4-helpernode](https://github.com/RedHatOfficial/ocp4-helpernode) playbook."

**Review by bpradipt — APPROVED** [review]
/lgtm

Thanks @yussufsh
